ui/views.py

```diff
@@ -280,3 +281,33 @@ def serve_media(request, media_path):
     response['Content-Length'] = os.path.getsize(file_path)
     response['Content-Disposition'] = "attachment; filename=%s" % filename
     return response
+
+
+@login_required
```

Comment:
Does this have (or need) any additional permissions beyond login, to prevent users from downloading others' exports?